docs/build-integration.md

```diff
@@ -0,0 +1,37 @@
+---
+id: build-integration
+title: Polkadot Integration Guide
+sidebar_label: Getting Started
```

Comment:
There's another sidebar label that is also 'Getting Started', could we maybe change this one so they aren't the same?

Reply:
Yes, but this one is clearly nested under the Integration Guide:
![image](https://user-images.githubusercontent.com/25483142/92085483-2d279680-edc9-11ea-8fdb-54d9aebe373c.png)

Reply:
Hm I see there. Personally I'd give it a different name but it should be alright if it's nested !

Reply:
I'm open to suggestions

Reply:
What about 'Getting started with Integrating', 'Integration Getting Started', or even 'Get Started with Integration'? :)